### ISSUE TITLE
Getting error while importing dropbox

### ISSUE THREAD

**Issue by prakashpp** [dropbox/dropbox-sdk-python#104]
```
Python 2.7.12 (default, Nov 19 2016, 06:48:10)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import dropbox
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/dropbox/__init__.py", line 3, in <module>
    from .dropbox import __version__, Dropbox, DropboxTeam, create_session  # noqa: F401
  File "/usr/local/lib/python2.7/dist-packages/dropbox/dropbox.py", line 35, in <module>
    from .session import (
  File "/usr/local/lib/python2.7/dist-packages/dropbox/session.py", line 7, in <module>
    from requests.packages.urllib3.poolmanager import PoolManager
ImportError: No module named packages.urllib3.poolmanager
>>>
```

I am getting above error while importing dropbox. Followings are the dependency versions I have installed -
chardet-3.0.3
dropbox-7.3.1
idna-2.5
requests-2.16.0
six-1.10.0
urllib3-1.21.1

Am I missing something?

**Comment by prakashpp** [comment]
found the issue, its the requests-2.16.0. Opened up #105, let me know if this any good.

**Comment by greg-db** [comment]
Thanks for the report! It looks like this was caused by requests 2.16.0 and fixed in 2.16.2.

For reference: 
- https://github.com/requests/requests/issues/4069
- https://github.com/requests/requests/issues/4073
